spec.bs

```diff
@@ -3065,17 +3058,21 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
   :: A [=list=] of interest group owner and interest group name pairs that bid
      in this auction.
   : <dfn>score</dfn>
-  :: {{double}} or Null. TODO
+  :: {{double}} or Null. For component auctions, contains the desirability
+     for the winning bid in the component auction.
   : <dfn>bid</dfn>
-  :: {{double}} or Null. TODO.
+  :: {{double}} or Null. For component auctions, contains the modified bid
+     for the winning bid in the component auction.
   : <dfn>bid currency</dfn>
-  :: [=currency tag=] or Null. TODO.
+  :: [=currency tag=] or Null. For component auctions, contains the currency of
+     the modified bid for the winning bid in the component auction.
   : <dfn>top level seller</dfn>
   :: An [=origin=] or Null. Null when the server response is not feeding into an
      on-device auction. Otherwise the value is the expected top-level seller for
      that auction.
   : ad metadata
-  :: A JSON [=string=]. TODO
+  :: A JSON [=string=]. For component auctions, contains the ad metadata

```

Comment:
1. this field's value goes to [[=generated bid/ad=]](https://github.com/WICG/turtledove/pull/1276/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR2824) at line 2824 above.
So we can define this field as `<dfn>ad metadata</dfn>` and reference it there?

2. is this supposed to be the winning bid's IG's [ad metadata](https://wicg.github.io/turtledove/#interest-group-ad-metadata)? If so, link to that.

3. Is it needed to specifically call out component auction? I.e., is it unclear to readers if we don't add this sentence?

Reply:
Clarified that this is the return value from scoring the component auction on the server.